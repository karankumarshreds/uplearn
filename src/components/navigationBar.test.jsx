import { render, screen } from "@testing-library/react";
import NavigationBar from "./navigationBar";

test("Should have user info in navigation bar", () => {
  const { container } = render(<NavigationBar />);
  expect(container).toMatchSnapshot();
});
