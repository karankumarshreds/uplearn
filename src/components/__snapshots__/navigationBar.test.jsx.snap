// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have user info in navigation bar 1`] = `
<div>
  <nav
    class="section"
  >
    <img
      alt=""
      class="logo"
      src="logo.jpg"
    />
  </nav>
</div>
`;
